src/pg/pg-store.ts

```diff
@@ -317,4 +319,20 @@ export class PgStore extends BasePgStore {
       results: results ?? [],
     };
   }
+
+  async getInscriptionCountPerBlock(): Promise<ValuesRowList<DbInscriptionCountPerBlock[]>> {
+    return await this.sql<DbInscriptionCountPerBlock[]>`
+      SELECT t1.block_height,
+        COUNT(*) FILTER (WHERE t2.genesis = true) AS count,
+        SUM(COUNT(*) FILTER (WHERE t2.genesis = true)) OVER (ORDER BY t1.block_height) AS scan_count
```

Comment:
I think it would be better to keep a table that accumulates these counts so we can perform these queries much faster (counts will get very heavy once the tables start growing larger).

The ingestion process needs to be adjusted for this to happen. We could do like a materialized view that we refresh with every new bitcoin block or something like that.